### ISSUE TITLE
easyaudit relies on django admin LogEntry (breaks without)

### ISSUE THREAD

**Issue by yapici** [soynatan/django-easy-audit#104]
When I run `python manage.py migrate easyaudit` I get this error:

`RuntimeError: Model class django.contrib.admin.models.LogEntry doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.`

Full stack:
```
Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/yapici/anaconda3/envs/py3/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/Users/yapici/anaconda3/envs/py3/lib/python3.7/site-packages/django/core/management/__init__.py", line 357, in execute
    django.setup()
  File "/Users/yapici/anaconda3/envs/py3/lib/python3.7/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/yapici/anaconda3/envs/py3/lib/python3.7/site-packages/django/apps/registry.py", line 120, in populate
    app_config.ready()
  File "/Users/yapici/anaconda3/envs/py3/lib/python3.7/site-packages/easyaudit/apps.py", line 8, in ready
    from easyaudit.signals import auth_signals, model_signals, request_signals
  File "/Users/yapici/anaconda3/envs/py3/lib/python3.7/site-packages/easyaudit/signals/auth_signals.py", line 6, in <module>
    from easyaudit.settings import REMOTE_ADDR_HEADER, WATCH_AUTH_EVENTS
  File "/Users/yapici/anaconda3/envs/py3/lib/python3.7/site-packages/easyaudit/settings.py", line 5, in <module>
    from django.contrib.admin.models import LogEntry
  File "/Users/yapici/anaconda3/envs/py3/lib/python3.7/site-packages/django/contrib/admin/models.py", line 39, in <module>
    class LogEntry(models.Model):
  File "/Users/yapici/anaconda3/envs/py3/lib/python3.7/site-packages/django/db/models/base.py", line 95, in __new__
    "INSTALLED_APPS." % (module, name)
RuntimeError: Model class django.contrib.admin.models.LogEntry doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```

**Comment by jheld** [comment]
Curious, have you removed django's admin app from your installed apps?

Also, which version of django?

**Comment by yapici** [comment]
Yes, I removed the admin app.

I'm using Django version 2.1.4.

**Comment by jheld** [comment]
Ok that's why. I'm not sure at this time why we rely on LogEvent. Don't have the code in front of me. But yes, a bug one way or the other in some form.

**Comment by jonathan-s** [comment]
Doing a quick search for logentry, and the only thing that shows up is https://github.com/soynatan/django-easy-audit/blob/03e05bc94beb29fc3e4ff86e313a6fef4b766b4b/easyaudit/settings.py#L43 

So at first glance it looks like it shouldn't depend on admin.

**Comment by tomdoel** [comment]
I'm getting a similar error when removing the admin.
Looks like this might already be fixed on master (858403b), it's just the fix isn't in the currently released PyPI version (1.1).

**Comment by jheld** [comment]
If this issue still persists after you upgrade, please re-open!
